default.nix

```diff
@@ -64,6 +64,7 @@
 with pkgs.lib;
 
 let
+  comp = f: g: (v: f(g v));
```

Comment:
Please put library functions in `lib.nix`.

Reply:
done